src/js/lib/admin-page/components/WipeMetadataCache.js

```diff
@@ -0,0 +1,20 @@
+import { __, sprintf } from '@wordpress/i18n';
+
+const WipeMetadataCache = ( { setting, label, note, onClick } ) => {
+	return (
+		<div className="container">
+			<div className="wipe-metadata-container">
+				<label className="wipe-metadata-label">{ __( label, 'wp-parsely' ) }</label>
+				<div className="wipe-metadata-controls">
+					<button type="button" className="wipe-metadata-button" onClick={ () => onClick( true ) }>{ __( 'Wipe your metadata?', 'wp-parsely' ) }</button>
```

Comment:
```suggestion
					<button type="button" className="wipe-metadata-button" onClick={ () => onClick( true ) }>{ __( 'Wipe Parse.ly plugin metadata?', 'wp-parsely' ) }</button>
```

Also, what benefit does the onClick attribute value provide here?